wweval/R/ms_fig4.R

```diff
@@ -484,14 +486,78 @@ make_fig4_rel_crps_by_phase <- function(scores) {
     geom_hline(aes(yintercept = 1), linetype = "dashed") +
     xlab("Epidemic phase") +
     ylab("Relative CRPS") +
-    scale_y_continuous(trans = "log10") +
+    scale_y_continuous(trans = "log10", limits = c(0.5, 2)) +
     get_plot_theme(
       x_axis_title_size = 8,
       y_axis_title_size = 8
     ) +
-    scale_fill_brewer(palette = "Set3")
+    scale_fill_manual(values = colors$phase_colors)
+
+  return(p)
+}
+#' Make figure that plots distribution of crps scores stratified by
+#' epidemic phase
```

Comment:
```suggestion
#' epidemic phase but aggregated across horizons
```

Reply:
wait, these are not aggregated across horizons. They are just plots of the distribution of scores for each loc-forecast-date-day combination, without any summarizing across the horizon